src/lib/helpers/complete-icon-set-helper.spec.ts

```diff
@@ -19,4 +19,27 @@ describe('Complete Iconset-helper', () => {
     const generatedContent = generateCompleteIconSetContent(fileNamesWithDefinitions, completeIconSetName);
     expect(unformatedString(expectedContent)).toEqual(unformatedString(generatedContent));
   });
+  it('should add interface when specified', () => {
+    let completeIconSetName = 'all-icons';
+    const fileNamesWithDefinitions = [
```

Comment:
Here, it would be great to have a test that tests that the interface is not generated if the types are turned off.